### PR TITLE
Issue #66 - show error if Perfherder Container has missing data

### PR DESCRIPTION
**Pull request by aimenbatool**
Hi @sarah-clements, @djmitche Please reviwe this PR for issue #66 .
This PR is in progress I have pushed it to get feedback on it. 
A few things to discuss: 

- First of all @armenzg  required an error panel if data is missing for more than 3 days but I have set the [condition for 1 day](https://github.com/aimenbatool/firefox-health-dashboard/blob/showErrorPanel-for-missing-data/src/components/ChartJsWrapper/index.jsx#L40url) just to display the error panel as we currently don't have data missing for 3 days.  I will update the conditon once we discuss the error panel. 

- I have used the error panel suggested by Armen, ( deploy and see at the end ) but i think we need to play with css. It is not looking nice or may be we can change poisition or use some simple error msg with custom css or anything else to make it look better. Suggestions required. 

- I need suggestion for [this piece of code](https://github.com/aimenbatool/firefox-health-dashboard/blob/showErrorPanel-for-missing-data/src/components/ChartJsWrapper/index.jsx#L33-L42) is there any better way of writing this? or is it fine? 

Thanks!

### CODE REVIEW

**Comment by sarah-clements** [comment]
The ErrorPanel component is imported from a library (mozilla-frontend-infra/components), so you probably won't be able to customize it too much. I'd suggest looking at the library documentation to see whether you can pass a class name as a prop. As far as positioning, maybe it'd look better if it was above the graphs.

**Comment by aimenbatool** [comment]
Hi @eliperelman, does this PR looks good to you or we need some changes?

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "@fortawesome/react-fontawesome": "^0.1.3",
     "@material-ui/core": "^1.3.1",
     "@material-ui/icons": "^3.0.1",
+    "@mozilla-frontend-infra/components": "^2.2.1",
```
[the remark this answers is not in the document]
Comment (eliperelman) [reply package.json]: The version numbers don't align between `components`, `icons`, and `core`. I think `core` and `icons` should be in the `^3.0.0` range to go along with `components`.

[the remark this answers is not in the document]
Comment (aimenbatool) [reply package.json]: I have upgraded both libraries.

**Comment by aimenbatool** [comment]
@sarah-clements thanks. 
Should we keep only three commits?

Issue #66 - show error if Perfherder Container has missing data
Upgrade @material-ui/core to v3.9.1
Upgrade @material-ui/icons to v3.0.2

Do you want me to merge?

**Comment by klahnakoski** [comment]
@aimenbatool What is the state of this PR?  Do you want me to merge it?

**Comment by aimenbatool** [comment]
@klahnakoski Yes, it is ready to merge. I just need suggestion about the commit. 

https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/267#issuecomment-460634407

**Comment by sarah-clements** [comment]
> @sarah-clements thanks.
> Should we keep only three commits?
> 
> Issue #66 - show error if Perfherder Container has missing data
> Upgrade @material-ui/core to v3.9.1
> Upgrade @material-ui/icons to v3.0.2
> 
You can squash it but include a brief summary of what changes you made (if you need guidance, look at [this guide ](https://github.com/erlang/otp/wiki/writing-good-commit-messages) to writing good commit messages).

**Pull request merged**